v22.2/cluster-setup-troubleshooting.md

```diff
@@ -304,6 +304,30 @@ The solution is to add connection retry logic to your application.
 
 ## Clock sync issues
 
+#### A node's timezone data has changed
+
+CockroachDB relies on each node's operating system timezone database to resolve named timezones like `America/New_York`, and for applying daylight saving rules. This can lead to inconsistent results across a cluster when coercing date/time strings into temporal types.
+
+Ensure that each node's operating system uses the same timezone database. In general, Cockroach Labs recommends that each node has an identical hardware and software configuration.
+
+Timezone data is read from the following sources in order of preference:
+
+1. The `ZONEINFO` operating system environment variable, if it is set to point to a `zoneinfo.zip` file such as the one that is included in Go.
+1. The operating system's timezone database, sometimes called `tzdata` or `zoneinfo`.
+
+    {{site.data.alerts.callout_info}}
+    CockroachDB on Windows does not use the operating system's timezone database.
```

Comment:
This callout is not exactly correct. on windows there is a tz database, but it has a different format and is relatively incomplete wrt crdb's needs.

Reply:
So where do we get our ZONEINFO in CRDB for Windows? Do we require the user to download a `zoneinfo.zip` or do we include it in the Windows installer or something else?

Reply:
We raise our hands up in the air and let them figure it out, because we don't really support windows anyways...